exercises/practice/hello-world/.meta/tests.toml

```diff
@@ -0,0 +1,4 @@
+[canonical-tests]
+
+# Say Hi!
+"af9ffe10-dc13-42d8-a742-e7bdafac449d" = true
```

Comment:
We're updating the spec for the `tests.toml` files: https://github.com/exercism/configlet/issues/198

Unfortunately, `configlet` hasn't yet been updated, so this is manual work at the moment :(

Reply:
Okay, I updated the tests.toml file to the new specification.